rhv_checks.py

```diff
@@ -42,7 +44,7 @@ def check_vm_count(system, warn=20, crit=30, **kwargs):
 
 def check_storage_domain_status(system, **kwargs):
     """ Check the usage of all the datastores on the host. """
-    okay, warning, critical, unknown, all = [], [], [], [], []
+    okay, warning, critical, unknown, over_all = [], [], [], [], []
```

Comment:
:tada: for changing over_all

Reply:
I know I said `over_all` was good on the call, but the more I think about it, it think makes more sense to call this `all_items`. What do you think @sbulage? `over_all` is really describing what the list contains.